scripts/sanitycheck

```diff
@@ -703,6 +719,7 @@ class DeviceHandler(Handler):
             command = [get_generator()[0], "-C", self.build_dir, "flash"]
 
         while not self.device_is_available(self.instance.platform.name):
+            logger.debug("Waiting for device {} to become available".format(self.instance.platform.name))
```

Comment:
I think better to remove that `logger` and `while not` loop. It will run only during the first flashing process, because 2nd and next flashing processes will have device available, and that loop will not run. I tested your sanitycheck without 721-723 strings -it works as usual, so I doubt if necessary to have that logger.
I used Microchip board to test your script, and board is by default is in `available status`, but when the first test is flashing into it, I see that printout in DEBUG, when 2nd test and next tests are flashing, I don't see that printout `DEBUG - Waiting for device mec15xxevb_assy6853 to become available`

Reply:
this is just a debug log message, it does not impact functionality. It means the device is locked and being used to run a test, so the next test can't execute yet. Useful for debugging possible lockups and races.

Reply:
> "Device programming. Soon it will be available" for example

the message is not related to flashing at all, the device is just not available and anything might be going on, for example a test might be running.

Reply:
Ok, understand. But in my case that message is printing during the first flashing process. And then test start to execute. It means that message is printing while the flashing process is running, but after second flashing process that message is not visible.